src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/UnitTestingIncrementalAnalyzerProviderWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api;
+using Microsoft.CodeAnalysis.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting
+{
+    internal class UnitTestingIncrementalAnalyzerProviderWrapper : IIncrementalAnalyzerProvider
```

Comment:
This one doesn't need the wrapper suffix (it's not exposing a Roslyn type to partner code)

```suggestion
    internal class UnitTestingIncrementalAnalyzerProvider : IIncrementalAnalyzerProvider
```